recipes/libboxes/all/conanfile.py

```diff
@@ -0,0 +1,97 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import copy, export_conandata_patches, get, rmdir
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout, CMakeDeps
+from conan.tools.scm import Version
+import os
+
+class libboxesRecipe(ConanFile):
+    name = "libboxes"
+    description = "Boxes is a set of frequently used containers built on top of STL"
+
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://boxes-twdev-projects-ba06c75b02a9b24f40d3a6b9b7130a3266c6a3c532.gitlab.io/"
+
+    topics = ("container", "utility")
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+
+    @property
+    def _min_cppstd(self):
+        return 20
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "apple-clang": "13",
+            "clang": "14",
+            "gcc": "11",
+            "msvc": "192",
+            "Visual Studio": "16.2",
```

Comment:
```suggestion
            "Visual Studio": "16",
```
You will have to use
```python
def loose_lt_semver(v1, v2):
    return all(int(p1) < int(p2) for p1, p2 in zip(str(v1).split("."), str(v2).split(".")))
```
for the version check to not exclude compler.version=16 otherwise.

Reply:
Oh, you can drop VS and MSVC from this list entirely since you don't support Windows anyway.

Reply:
Removed. Thanks